package/src/renderer/components/backdrop/BackdropFilter.tsx

```diff
@@ -13,7 +13,7 @@ export const BackdropFilter = ({
   filter,
   children: groupChildren,
   ...props
-}: SkiaProps<BackdropFilterProps>) => {
+}: SkiaProps<BackdropFilterProps, "filter">) => {
```

Comment:
Why do we add "filter" here? I see that we have added an O for the SkiaProps but not sure if I get why?

Reply:
with this upgrade, it complains about "animated nodes" which is a good thing, shows that types are more precise.